TicTacToe/ViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  ViewController.swift
+//  TicTacToe
+//
+//  Created by Tahia Ata on 5/16/16.
+//  Copyright © 2016 Tahia Ata. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    @IBOutlet weak var commentLabel: UILabel!
+    @IBOutlet weak var ticCollectionView: UICollectionView!
+    @IBOutlet weak var firstPlayerLabel: UILabel!
+    @IBOutlet weak var secondPlayerLabel: UILabel!
+
+    var ticArray = [UIImage?](count: Constants.gameInfo.totalGridNumber, repeatedValue: UIImage(named:Constants.gameInfo.defaultImageName))
+    let firstPlayer = Constants.PlayerInfo.playerOne
+    let secondPlayer = Constants.PlayerInfo.playerTwo
+    var firstPlayerName = Constants.PlayerInfo.firstPlayerName
+    var secondPlayerName = Constants.PlayerInfo.secondPlayerName
+    var currentPlayer = Constants.PlayerInfo.playerOne
+    var count = 0
+    var checkArray : [[Int]] = [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
+    var win = false
+    let numberOfRowColumn = 3
```

Comment:
Howerver, in case data type is implicit, specification is necessary.

// ◯ Int / UInt / NSInteger / NSUInteger these data types should be distinguished so specification is necessary
let i: Int = 0

// ◯ Float / CGFloat / Double these data types should be distinguished so specification is necessary
let p: Float = 3.14